### DISCUSSION TITLE
Github Pages and PhPStorm Previews differ.

### DISCUSSION THREAD

**Discussion by lsara011** [community/community#60207] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey everyone!

my name is Luis. I am a novice web developer based in Miami, Florida and would like some help. 
I am currently using PhP storm to build my website. When previewed through Chrome, it looks perfectly. All the links and headers are in place where I want them to be. But when previewed through Github Pages, it looks horrible. Its missing a lot of elements and everything looks like a mess despite looking really well through PhpStorm. Does anyone know why or what I can do?

**Comment by GhostOf0days** [comment, marked as answer]
Github Pages serves static websites. If your website is dynamic, that may be why. I suggest not using Github Pages. Instead, use Netlify, Heroku, or Vercel. Use Netlify for simple use cases/websites and Vercel for complex use cases/websites. As you are a novice, before you get deeper into it, I also suggest you not to use PHP for web dev. It is an outdated language. Use a modern JavaScript tech stack using things like React and Next.js. Even better would be using TypeScript.

**Reply by lsara011** [reply]
I appreciate the response! Ill definitely look into those. Thank you!
